ios/src/platform_ios.mm

```diff
@@ -57,12 +58,29 @@ bool isContinuousRendering() {
     
 }
 
+std::string setResourceRoot(const char* _path) {
+
+    NSString* path = [NSString stringWithUTF8String:_path];
+
+    if (*_path != '/') {
+        NSString* resources = [[NSBundle mainBundle] resourcePath];
+        path = [resources stringByAppendingPathComponent:path];
+    }
+
+    s_resourceRoot = [path stringByDeletingLastPathComponent];
```

Comment:
We only want `s_resourceRoot` to contain the bundle resource path if a relative path is used, i.e. if `_path` does not begin with a `/`.